cmd/docs.go

```diff
@@ -0,0 +1,102 @@
+package cmd
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/TouchBistro/goutils/fatal"
+	"github.com/TouchBistro/tb/config"
+	"github.com/TouchBistro/tb/docker"
+	"github.com/TouchBistro/tb/util"
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+var docsCmd = &cobra.Command{
+	Use:   "docs <service-name> (Experimental)",
```

Comment:
`(Experimental)` needs to go on `Short` and `Long` for it to should up in the docs properly.